src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -382,24 +381,37 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     }
   }
 
-  private showModal(modalType: ModalType = ModalType.ENROLL): void {
+  private showModal(modalType: ModalType = ModalType.ENROLL, hideSecondary = false): void {
     // Check if config is defined and timer is available
     const isSnoozeTimeAvailable = this.config?.timer.isSnoozeTimeAvailable() ?? false;
 
     // Show the modal with the provided modal type
     const {modalOptions, modalType: determinedModalType} = getModalOptions({
-      hideSecondary: !isSnoozeTimeAvailable,
-      primaryActionFn: () => this.enroll(),
+      hideSecondary: !isSnoozeTimeAvailable || hideSecondary,
+      primaryActionFn: () => {
+        if (modalType === ModalType.SNOOZE_REMINDER) {
```

Comment:
What is this for? Should we not hide the primary button if it is not meant to do anything?

Reply:
in the mock we have a ok button, on click of that we close the modal.

<img width="492" alt="Screenshot 2023-12-22 at 13 55 42" src="https://github.com/wireapp/wire-webapp/assets/28754444/8a1dc4fe-522d-4571-b44b-74c9954e10ae">